src/main/webapp/assets-lti/js/factories/factories.js

```diff
@@ -114,8 +114,15 @@ canvasSupportApp.factory('Course', function ($http) {
 //SECTIONS FACTORY - does the request for the sections controller
 canvasSupportApp.factory('Sections', function ($http) {
   return {
-    getSectionsForCourseId: function (courseId) {
-      var url = '/canvasCourseManager/manager/api/v1/courses/' + courseId + '/sections?per_page=100&_='+ generateCurrentTimestamp();
+    getSectionsForCourseId: function (courseId, context) {
+    	console.log('Course: ' + courseId);
```

Comment:
Should remove console.log. Will result in errors if browser does not have a console (like IE). We should have used Angular's $log wrapper, too late now.